workflows/chipseq/chipseq_trackhub.py

```diff
@@ -144,28 +146,30 @@ def decide_color(samplename):
         sample_dir, sample,
         sample + '.cutadapt.unique.nodups.bam.bigwig')
 
-    subgroup = df[df.loc[:, 'label'] == sample].to_dict('records')[0]
-    subgroup = {
-        sanitize(k, strict=True): sanitize(v, strict=True)
-        for k, v in subgroup.items()
-    }
-    subgroup['algorithm'] = 'NA'
-    subgroup['peaks'] = 'no'
-
-    signal_view.add_tracks(
-        Track(
-            name=sanitize(sample + os.path.basename(bigwig), strict=True),
-            short_label=sample,
-            long_label=sample,
-            tracktype='bigWig',
-            subgroups=subgroup,
-            source=bigwig,
-            color=decide_color(sample),
-            altColor=decide_color(sample),
-            maxHeightPixels='8:35:100',
-            viewLimits='0:500',
+    if os.path.isfile(bigwig):
```

Comment:
was this complaining on symlinks? or directories?

Reply:
I honestly don't know what was up with this one, it must have drawn my attention somehow but I recall being tired when I was coding this and cannot remember what the issue was. I apologize and can remove it if you prefer.

Reply:
Reverted.